### ISSUE TITLE
Allowing normal users to create groups and categories and manage those groups and categories themselves? like sub reddits.

### ISSUE THREAD

**Issue by FaizanZahid** [yariplus/nodebb-plugin-modmin#2]
https://community.nodebb.org/topic/13469/bounty-of-200-plugin-that-allows-mods-to-create-and-manage-sub-categories-and-groups/8

Will this serve this purpose?

Great plugin btw!

**Comment by yariplus** [comment]
Yes, to some extent. Although, there is still room for improvement.

You would be able to do this by giving the "registered-users" group the "Manage Category" and "Assign Groups" privileges in the "Global" category. They would then be able to create categories which are completely isolated from any users not in their assigned group.
